Lib/bdb.py

```diff
@@ -323,6 +438,10 @@ def clear_all_file_breaks(self, filename):
         del self.breaks[filename]
```

Comment:
When there is an explicit non-None return in a function, then any None return should be explicit.  (This is somewhere in PEP 8.)  So add 'return None' on new line after this.